sphinx/cmd/quickstart.py

```diff
@@ -417,7 +409,8 @@ def generate(d, overwrite=True, silent=False, templatedir=None):
     for name in EXTENSIONS:
         if d.get('ext_' + name):
             d['extensions'].append('sphinx.ext.' + name)
-    d['extensions'] = (',\n' + indent).join(repr(name) for name in d['extensions'])
+    d['extensions'] = (',\n' + indent).join(
+        repr(name) for name in d['extensions'])
```

Comment:
It seems not changed here.

Reply:
That's because we know for a fact that `d['extensions']` will exist, thus, we don't need to use `.get` notation here